include/InstrumentPlayHandle.h

```diff
@@ -56,13 +56,13 @@ class EXPORT InstrumentPlayHandle : public PlayHandle
 		do
 		{
 			nphsLeft = false;
-			foreach( const NotePlayHandle * cnph, nphv )
+			for( const NotePlayHandle * constNotePlayHandle : nphv )
 			{
-				NotePlayHandle * nph = const_cast<NotePlayHandle *>( cnph );
-				if( nph->state() != ThreadableJob::Done && ! nph->isFinished() )
+				NotePlayHandle * notePlayHandle = const_cast<NotePlayHandle *>( constNotePlayHandle );
```

Comment:
Because `NotePlayHandle::nphsOfInstrumentTrack()` returns a `ConstNotePlayHandleList`, but `notePlayHandle->process()` is invalid for const NotePlayHandles. But as I told michaelgregorius, "that part of InstrumentPlayHandle.h should be removed or otherwise fixed in the near future anyway per #2606 (on which feedback is still welcome, btw)."